### ISSUE TITLE
augmentedRCBD for Multi-environment trials

### ISSUE THREAD

**Issue by myaseen208** [aravind-j/augmentedRCBD#1]
Congratulations for developing such a wonderful package. I wonder if there is any plan to add the facility to analyze augmented RCBD performed across different environments. Thanks

**Comment by Allan-gitrepos** [comment]
Yes, adding the facility to analyze multi-seasonal or multi-environmental data would be wonderful and literally completes the package. This will be useful most conservationist like me.

**Comment by hdonoho2** [comment]
Agreed! It would be very useful to have the option to add multiple environments into the ANOVA with this package. :)
